algorithms/kernel/k_nearest_neighbors/kdtree_knn_impl.i

```diff
@@ -83,75 +83,54 @@ class Stack
 public:
     Stack() : _data(nullptr) {}
 
-    ~Stack()
```

Comment:
Coding guidelines violation (declaration order)

Reply:
It was fixed